pyproject.toml

```diff
@@ -17,6 +17,8 @@ classifiers = [
     "Programming Language :: Python :: 3",
```

Comment:
This pyproject.toml file replaces the `setup.cfg` and `setup.py` files. This reduces the number of files, centralizes project configuration and is the way forward for modern Python packaging.